dali/operators/generic/reshape.h

```diff
@@ -40,6 +41,11 @@ class Reshape : public Operator<Backend> {
   bool SetupImpl(std::vector<OutputDesc> &output_desc, const Workspace &ws) override;
 
   void RunImpl(Workspace &ws) override;
+ 
+ protected:
+  virtual void CalculateOutputShape(const Workspace &ws);
+  std::vector<int> src_dims_;
+  void CheckSrcDims(const Workspace &ws);
```

Comment:
```suggestion
  virtual void CalculateOutputShape(const Workspace &ws);
  void CheckSrcDims(const Workspace &ws);

  std::vector<int> src_dims_;
```
Nitpick: Maybe group functions and member variables?